t/t1092-sparse-checkout-compatibility.sh

```diff
@@ -912,7 +912,7 @@ test_expect_success 'read-tree --prefix' '
 	test_all_match git read-tree --prefix=deep/deeper1/deepest -u deepest &&
 	test_all_match git status --porcelain=v2 &&
 
-	test_all_match git rm -rf --sparse folder1/ &&
+	run_on_all git rm -rf --sparse folder1/ &&
```

Comment:
It's probably worth trying to submit upstream without the change, as long as we explain that it has some precedent _and_ it could be fixed in the future without expanding the index.

Reply:
I'm thinking the same. So I will submit this without tree-traversal change first.